### PR TITLE
Fix list election mobile layout

### LINKED ISSUE #4358: List Election: two issues
**Describe the bug**
Two errors occur in a list election:

1) 1) Content error: The display of the number of candidates is incorrect in one place. Instead of displaying the correct number of candidates, a '1' is displayed. I assume that the '1' has a reference to the title under the election options and is therefore read out as '1'. As you see in the image beneath, the candidate count doesn't match. The number in both red squares should match. The correct number is the lower one. 

image

2) The layout does not fit in the mobile view. The title 'Confirmation of the nomination list' is between the election option texts and the election option buttons. This does not fit. In this case, the text must be above the voting option texts (yes/no/abstain).

Red square marks the relevant text, which needs to be replaced.
image

**How to Reproduce**
1. open a meeting
2. open elections
3. create an election > add two candidates > create a ballot > voting method: yes/no/abstain per list; voting groups: admin; save > start ballot
4. For first problem: You should see now the wrong numbers just like in the first image
5. For second problem: open the dev-console and change your view to mobile > now you should see the problem

**Expected behavior**
- [x] First problem: The numbers in the red square in the first image should match. The reference is the lower number, which refers to the number of candidates on the list
- [x] Second problem: The text 'Confirmation of the nomination list' should be placed right above the voting options.

### PR DESCRIPTION
**Pull request by anbebe**
Closes #4358

### CODE REVIEW

File: client/src/app/site/pages/meetings/modules/poll/components/base-poll-vote/base-poll-vote.component.html [hunk 1]
```diff
@@ -73,6 +73,11 @@ <h4 class="poll-delegation-title">
                                     'yna-grid':
                                         poll.isMethodYNA ||
                                         (settings.isSplitSingleOption && displayed_in_autopilot && poll.isMethodYNA),
+                                    'yna-grid-extended':
+                                        poll.isMethodYNA &&
+                                        isMobile &&
+                                        !settings.isSplitSingleOption &&
+                                        poll.options[0].isListOption,
```
Comment (Elblinator) [inline client/src/app/site/pages/meetings/modules/poll/components/base-poll-vote/base-poll-vote.component.html:80]: - please remove isSplitSingleOption 
- change poll.options... to poll.isListPoll
- Add !isMobile to yna-grid

File: client/src/app/site/pages/meetings/modules/poll/components/base-poll-vote/base-poll-vote.component.html [hunk 2]
```diff
@@ -92,9 +97,23 @@ <h4 class="poll-delegation-title">
                                     !(settings.isSplitSingleOption && displayed_in_autopilot)
                                 ) {
                                     <b class="grid-name-area">
-                                        {{ poll.option_ids.length }} {{ optionPluralLabel | translate }}
+                                        @if (poll.options[0].isListOption) {
```
Comment (Elblinator) [inline client/src/app/site/pages/meetings/modules/poll/components/base-poll-vote/base-poll-vote.component.html:100]: poll.isListPoll

File: client/src/app/site/pages/meetings/modules/poll/components/base-poll-vote/base-poll-vote.component.html [hunk 3]
```diff
@@ -92,9 +97,23 @@ <h4 class="poll-delegation-title">
...
+                                            {{ poll.options[0]?.contentTitlesAsSortedArray.length }}
+                                        } @else {
+                                            {{ poll.option_ids.length }}
+                                        }
+                                        {{ optionPluralLabel | translate }}
                                     </b>
                                 }
+                                @if (!settings.isSplitSingleOption && poll.options[0].isListOption && isMobile) {
+                                    @if (poll.options[0].content_object) {
```
Comment (Elblinator) [inline client/src/app/site/pages/meetings/modules/poll/components/base-poll-vote/base-poll-vote.component.html:109]: - poll.isListPoll
- and poll.options[0]?.content_object

File: client/src/app/site/pages/meetings/pages/polls/view-models/view-option.ts [hunk 4]
```diff
@@ -30,6 +30,10 @@ export class ViewOption<C extends BaseViewModel = any> extends BaseViewModel<Opt
         return this.content_object;
     }
 
+    public get contentEntries(): number {
+        return (this.content_object as SortedList).entries.length;
+    }
+
```
Comment (Elblinator) [inline client/src/app/site/pages/meetings/pages/polls/view-models/view-option.ts:36]: Please remove unused code

**Review by Elblinator — CHANGES_REQUESTED** [review]

**Review by Elblinator — APPROVED** [review]
code review

**Review by bspekker — CHANGES_REQUESTED** [review]
Layout for voting method YNA per list looks good. 

New error for voting method YNA per candidate in the mobile layout: 
image
The YNA options are not above the buttons for the options. 
The caption of the buttons should always be directly above them.

**Review by Elblinator — APPROVED** [review]
code

**Review by bspekker — APPROVED** [review]

**Pull request merged**